javascript/javascript1/week2/Ali-Sharifi/script2.js

```diff
@@ -0,0 +1,126 @@
+//Formal fullname
+function getFullname(firstname, lastname, useFormalName = false, gender = "") {
+  firstname = firstname.trim();
```

Comment:
Please, set up [Code Spell Check](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) VS Code extension. It will help you write better variable names, and comments. In this case it would suggest you follow the camelCase naming convention:
```suggestion
  firstName = firstName.trim();
```